package.json

```diff
@@ -7,39 +7,7 @@
     "url": "https://github.com/webex/webex-js-sdk/issues"
   },
   "license": "MIT",
-  "contributors": [
-    "Adam Weeks <adweeks@cisco.com>",
-    "Aimmee Ma <aimma@cisco.com>",
-    "Alex Archer <alearche@cisco.com>",
-    "Andrew Holsted <holsted@cisco.com>",
-    "Arun Ganeshan <arungane@cisco.com>",
-    "Bernie Zang <nzang@cisco.com>",
-    "Brad Rydzewski <brydzews@cisco.com>",
-    "Brian Bender <brbender@cisco.com>",
-    "Carter Foxgrover <cfoxgrov@cisco.com>",
-    "Christopher DuBois <chdubois@cisco.com",
-    "David Greenstein <dagreens@cisco.com>",
-    "Fong Chin <fonchin@cisco.com>",
-    "Greg Hewett <ghewett@cisco.com>",
-    "Hans Steinbrecher <hsteinbr@cisco.com>",
-    "Ian W. Remmel <iremmel@cisco.com>",
-    "Joe Fuhrman <jofuhrma@cisco.com>",
-    "Joe Sankey <josankey@cisco.com>",
-    "Josh Dykstra <jodykstr@cisco.com>",
-    "Joshua Dykstra <jodykstr@cisco.com>",
-    "Matt Norris <matnorri@cisco.com> (http://mattnorris.me)",
-    "Michael Wegman <mwegman@cisco.com>",
-    "Nate Cohen-Smith <ncohensm@cisco.com>",
-    "Nick Clarke <nickclar@cisco.com>",
-    "Patrick Linskey <plinskey@cisco.com>",
-    "Riley Marsh <rimarsh@cisco.com>",
-    "Saurabh Jain <saurjai3@cisco.com>",
-    "Taymoor Khan <taykhan@cisco.com>",
-    "Tim Scheuering <timsch@cisco.com>",
-    "Tor Hagemann <tohagema@cisco.com>",
-    "Tran Tu <tratu@cisco.com>",
-    "Vagou Zhou <vagouz@cisco.com>"
-  ],
+  "author": "devsupport@webex.com",
```

Comment:
Glad we're changing to devsupport for author.

Reply:
No. I miss having my name in there. ;)

Reply:
haha, I'm just doing what we agreed a looooonnng time back 😬